controller/setting_controller.go

```diff
@@ -140,6 +140,14 @@ func NewSettingController(
 	return sc
 }
 
+func getLoggerForSetting(logger logrus.FieldLogger, setting *longhorn.Setting) *logrus.Entry {
```

Comment:
No need, because we don't need to handle setting resource uninstall. Just leave it to the normal resource clean up via CRD removing.

Reply:
removed